pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -333,6 +336,7 @@ func TestGameServerApplyDefaults(t *testing.T) {
 			assert.Equal(t, test.expected.scheduling, test.gameServer.Spec.Scheduling)
 			assert.Equal(t, test.expected.health, test.gameServer.Spec.Health)
 			assert.Equal(t, test.expected.sdkServer, test.gameServer.Spec.SdkServer)
+			assert.Equal(t, test.expected.alphaPlayerCapacity, test.gameServer.Status.Alpha.Players.Capacity)
```

Comment:
To my comment above, would it make more sense to verify that the status is not there at all unless we explicitly enable a feature flag?